### PR TITLE
Adds Dog D

### PR DESCRIPTION
**Pull request by joakimskoog**

### CODE REVIEW

File: data/dog_d.json [hunk 1]
```diff
@@ -0,0 +1,171 @@
+{
+  "name": "DOG D",
```
Comment (sammdec) [inline data/dog_d.json:2]: The other way around was great "Dog D"

File: data/dog_d.json [hunk 2]
```diff
@@ -0,0 +1,171 @@
+{
+  "name": "DOG D",
+  "tagline": "ANNIVERSARY IMPERIAL STOUT.",
```
Comment (sammdec) [inline data/dog_d.json:3]: Could you change this to title case "Anniversary Imperial Stout."

File: data/dog_d.json (hunk 1 above)
Comment (joakimskoog) [inline data/dog_d.json:2]: Alright then, give me a minute. :)

File: data/dog_d.json [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+      }
+    ],
+    "hops": [
+      {
+        "name": "Chinook",
+        "amount": {
+          "value": 50,
+          "unit": "grams"
+        },
+        "add": "Start",
```
Comment (sammdec) [inline data/dog_d.json:104]: Could you lowercase this word please, just to keep consistent with the other recipes :smiley:

File: data/dog_d.json [hunk 4]
```diff
@@ -0,0 +1,171 @@
...
+        "attribute": "Bitter"
```
Comment (sammdec) [inline data/dog_d.json:105]: Lowercase please

**Comment by sammdec** [comment]
Apart from these notes it all looks great :+1: Thanks so much

**Pull request closed without merging**